components/TreeHeatmap/index.js

```diff
@@ -177,7 +186,7 @@ export default class TreeHeatmap extends VisComponent {
     let colLinks = [];
     if (clusterColumns) {
       columns.forEach(col => {
-        if (+metadataRows._cluster[col.name] >= 0) {
+        if (metadataRows._cluster && +metadataRows._cluster[col.name] >= 0) {
```

Comment:
Similar thing here as for cast-to-string - I think best practice now is to use `Number()` instead of the unary `+` (but [citation needed]).